rustfmt.toml

```diff
@@ -1,3 +1,4 @@
 error_on_line_overflow = true
 error_on_unformatted = true
 style_edition = "2024"
+overflow_delimited_expr = false
```

Comment:
I added this to reduce churn, as the 2024 default (`true`) will require some updates to the formatting of the rustfmt codebase itself